### ISSUE TITLE
Add DeepCPMatte sphere or cube 3d preview

### ISSUE THREAD

**Issue by falkhofmann** [charlesangus/DeepC#83]
during some user questioning it came up as an idea to have optional display the sphere or cube in 3d context. in addition/swap to the axis handle.
perhaps with some sort of preview checkbox?

not seeing this as important, rather a nice to have in some scenarios.
therefore documenting the idea here.

**Comment by charlesangus** [comment]
Yeah, would definitely be helpful. Possibly easiest to implement as a gizmo
with a sphere in it, tbh, but should also be an interesting delve into the
mysteries of Nuke's handle drawing functionality.

**Comment by falkhofmann** [comment]
the handle approach would also be my favorite. i think in the nuke example code file is a sphere node to look at.

at the end, it might not be needed as detailed as this since the poly count/resolution is irrelevant, but the transformations need to be applied. also maybe it would be nicer if its not 100% opaque.

but maybe starting of simple is better.
